packages/e2e-tests/src/specs/editor/shopping/woocommerce.js

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+
+// eslint-disable-next-line import/no-extraneous-dependencies -- in package.json
+import { matchers } from 'jest-json-schema';
+
+import {
+  createNewStory,
+  withExperimentalFeatures,
+  publishStory,
+  activatePlugin,
+  deactivatePlugin,
+  takeSnapshot,
+} from '@web-stories-wp/e2e-test-utils';
+
+/**
+ * Internal dependencies
+ */
+import * as schema from './schema.json';
+
+// eslint-disable-next-line jest/require-hook -- matchers only for this set of tests
+expect.extend(matchers);
+
+describe('Shopping', () => {
+  describe('Shopping schema', () => {
+    const WPVersion = process.env?.WP_VERSION || 'latest';
+
+    withExperimentalFeatures(['shoppingIntegration']);
+
+    beforeAll(async () => {
+      if ('latest' === WPVersion) {
+        await activatePlugin('woocommerce');
+      }
+    });
+
+    afterAll(async () => {
+      if ('latest' === WPVersion) {
+        await deactivatePlugin('woocommerce');
+      }
+    });
+
+    if (WPVersion !== 'latest') {
+      //eslint-disable-next-line jest/expect-expect, jest/no-focused-tests
+      it.only('minimum WordPress requirement not met', () => {});
+    }
```

Comment:
Can use `minWPVersionRequired` from `e2e-utils` package.

Reply:
So ... I swapped this back to using minWPVersionRequired

The WP 5.5. e2e test run is failing as it's looking for a Plugin that doesn't exist --- any way we can avoid this outside of how I was doing it prior.


WP 5.5
```
No node found for selector: tr[data-slug="woocommerce"] .activate a

      29 |
      30 |   if (!disableLink) {
    > 31 |     await page.click(`tr[data-slug="${slug}"] .activate a`);
         |     ^
      32 |     await page.waitForSelector(`tr[data-slug="${slug}"] .deactivate a`);
      33 |   }
```